utils/cliutils/commandsflags.go

```diff
@@ -1650,7 +1650,7 @@ var flagsMap = map[string]cli.Flag{
 	},
 	lcSigningKey: cli.StringFlag{
 		Name:  SigningKey,
-		Usage: "[Mandatory] The GPG/RSA key-pair name given in Artifactory.` `",
+		Usage: "[Optional] The GPG/RSA key-pair name given in Artifactory. If not provided will create/use the default one` `",
```

Comment:
[Optional] The GPG/RSA key-pair name given in Artifactory. If not provided will create/use the default one
-->
[Optional] The GPG/RSA key-pair name given in Artifactory. If the key isn't provided, the command creates or uses the default key

Let's also have a PR with this change opened to [github.com/jfrog/documentation](https://github.com/jfrog/documentation) and link the PR to this PR by adding to the PR description. [github.com/jfrog/documentation](https://github.com/jfrog/documentation) includes the source files for JFrog CLI's online documentation.
We will merge this PR documentation once this functionality is released.